doc/nrf/libraries/bootloader/bl_storage.rst

```diff
@@ -8,7 +8,8 @@ Bootloader storage
    :depth: 2
 
 The bootloader storage library is used to read and write both one-time programmable (OTP) data and a non-volatile counter.
-It can be used by both the |NSIB| (NSIB) and the application, but the application has read access only.
+If the |NSIB| (NSIB) is used the library can only be used by the NSIB and must not included into the application image.
```

Comment:
If the |NSIB| (NSIB) is enabled it should be only user of this library. The application image can only include this library when |NSIB| (NSIB) is disabled.

Seems a bit more clear to me